spec/rails_helper.rb

```diff
@@ -9,6 +9,13 @@
 require File.join(File.dirname(__FILE__), 'support/helpers/registration')
 require File.join(File.dirname(__FILE__), 'support/helpers/authentication')
 require 'capybara/rspec'
+require 'database_cleaner'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.